applications/machine_learning/app_desc.rst

```diff
@@ -545,6 +575,33 @@ After programming the application, perform the following steps to test the nRF M
 Optionally, you can also connect to the device using `Edge Impulse's data forwarder`_ and forward data to `Edge Impulse studio`_ (after logging in).
 See `Forwarding data to Edge Impulse studio`_ for details.
 
+Testing with the nRF54L15 DK
+---------------------------------------
+
+After programming the application, perform the following steps to test the nRF Machine Learning application on the DK:
+
+1. Turn on the development kit.
+   The application starts in a mode that runs the machine learning model.
+   Initially, **LED3** displays the LED effect representing ``sine`` wave (1 blink), and **LED1** is breathing, because the signal was not yet recognized by the machine learning model.
```

Comment:
```suggestion
   Initially, **LED3** displays the LED effect representing ``sine`` wave (one blink), and **LED1** is breathing, because the signal was not yet recognized by the machine learning model.
```